samples/system-test/test_util.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const container = require('@google-cloud/container');
```

Comment:
would it be possible to just export the retry logic you have in the samples instead of creating this extra file? Will be confusing for maintainers in the future.

Reply:
This is only used in the test files and not in the samples itself. The retry logic here is different what is used in the samples. 

The retry logic in the samples is a `async function that is scheduled with setInterval`. Since the last operation in the sample is either creation or deletion, the program waits for the async function to resolve and all setIntervals to be done. 

However, in the test cases, we need to wait in the `before` method for the cluster to be setup completely - in order to do our tests. In this case if we use the same `async function` method in the samples, the tests start running just after running the `before` method. However, ideally we want to force the `before` to wait until the setup is complete. Thus, in this case we use a `sleep` to ensure we wait until the setup is done. So it is this custom logic for tests that is added in this seperate file. 

Let me know if it makes sense!

Reply:
I've added a description of this method here: https://github.com/googleapis/nodejs-cloud-container/pull/519/commits/057d11e6b64febd5c495083d79950e4d315a08c4